config/jobs/kubernetes/sig-testing/compatibility-versions-e2e.yaml

```diff
@@ -115,3 +115,53 @@ periodics:
           # this is mostly for building kubernetes
           memory: 9Gi
           cpu: 7
+- interval: 6h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-e2e-kind-compatibility-versions-feature-gate-test
+  annotations:
+    testgrid-dashboards: sig-testing-kind
+    testgrid-tab-name: compatibility-versions-feature-gate-test
+    description: Uses kind to run bespoke feature gate tests from the n-1 kubernetes release yaml files against a latest kubernetes master components w/ --emulated-version=n-1 set.
+    # TODO(aaron-prindle) route the alert email to a rotation vs individual email
+    testgrid-alert-email: aprindle@google.com
+    testgrid-num-columns-recent: '6'
```

Comment:
you may want to to tune testgrid-num-failures-to-alert to say, 2, as well, given that would be ~12h to alert, (1 may be annoying for flakes, OTOH release blocking jobs need to be kept non-flaky before introducing), 3 is the default which puts this at 18h before alerting currently.

Reply:
Ah makes sense. I've updated this test to have:

```yaml
 testgrid-num-failures-to-alert to say: 2
 ```